src/app/pages/CourseDetail/CourseDetail.module.css

```diff
@@ -0,0 +1,31 @@
+.container {
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto 1fr;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  width: 90%;
```

Comment:
Why don't you give the container a width with 90% if main and header are inside?

Reply:
Thank you. There is a button in the footer of the page (without CSS styling) that has a with of 100%.